docs/run_bmorph_template.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
This is just the notebook for the stuff above right? It doesn't seem to include all of the text but the code looks the same. So if you modify the notebook you'd have to rewrite all of the text? Could we just embed the text from the tutorial above into the notebook to sidestep this issue?

Reply:
The notebook and the text are not linked, so updating one will not impact the other. My thought behind the notebook as to have all the code of the tutorial without have to weed through all the text, or without having to copy+paste over the code from the tutorial. Was kind of thinking of the tutorial as the instructions and the notebook as the blank worksheet you can fill out.

Reply:
In my view the text *is* part of the tutorial. IMO it would be better to merge them together.

Reply:
Alright, I can incorporate the changes from the text tutorial into the notebook tutorial